module1.js

```diff
@@ -53,11 +44,15 @@ function question1(name1) {
   @returns {string} odd or even
 */
 
-// function question3(number1) {
-//   if (number1) {
-//     return;
-//   }
-// }
+function question3(number1) {
+  if (number1 % 2 === 0) {
+    return "even";
+  }
+    else {
+      return "odd";
+    }
+  }
+
 
```

Comment:
Good. You might want to look up ternary operators; they should be useful here, where there are only two possible answers.